rocky/tools/models.py

```diff
@@ -248,8 +248,11 @@ def has_perm(self, perm: str) -> bool:
     def has_perms(self, perm_list: Iterable[str]) -> bool:
         return all(self.has_perm(perm) for perm in perm_list)
 
-    def has_clearance(self, scan_level: int) -> bool:
-        return self.acknowledged_clearance_level >= scan_level
+    def has_clearance(self, plugin: Plugin) -> bool:
+        perm = self.has_perm("tools.can_enable_disable_boefje")
+        if hasattr(plugin, "scan_level"):
```

Comment:
It makes more sense to check whether the `plugin` instance is `Boefje`

```suggestion
        if isinstance(plugin, Boefje):
```